db/test_db_init/021_threads_insert.sql

```diff
@@ -220,3 +220,29 @@ INSERT INTO user_thread_identities(thread_id, user_id, identity_id)
       (SELECT id FROM Users WHERE username = 'SexyDaddy69'),
       (SELECT id FROM secret_identities WHERE display_name = 'The Prophet'));
  
+/* Insert test thread for deleting */
```

Comment:
See my comment at the end on the file. We may or may not want this. I'm inclined to trying to just delete existing ones for now, and then only do this if the db stuff I hacked in (which automatically restores DB state between tests) yells at us later.